nomad/resources_endpoint.go

```diff
@@ -73,6 +73,15 @@ func getResourceIter(context, prefix string, ws memdb.WatchSet, state *state.Sta
 	}
 }
 
+// If the length of a string is odd, return a subset of the string to the last
+// even character
+func roundDownIfOdd(s string) string {
```

Comment:
I am fine with this if it is a place holder to a slightly more advanced implementation. My concern here is two fold:

1) Jobs don't need to be rounded, only UUIDs.
2) If I do a prefix of ab1 and I get a return of: ["ab12..", "ab23...", "ab34", ...] something seems broken :)